frontend/javascripts/admin/task/task_create_form_view.js

```diff
@@ -390,11 +393,17 @@ class TaskCreateFormView extends React.PureComponent<Props, State> {
                   isEditingMode || this.state.specificationType === SpecificationEnum.BaseAnnotation
                 }
               >
-                {this.state.datasets.map((dataset: APIDataset) => (
-                  <Option key={dataset.name} value={dataset.name}>
-                    {dataset.name}
+                {this.state.isFetchingData ? (
+                  <Option value="Loading" disabled>
+                    Loading...
                   </Option>
-                ))}
+                ) : (
+                  this.state.datasets.map((dataset: APIDataset) => (
+                    <Option key={dataset.name} value={dataset.name}>
+                      {dataset.name}
+                    </Option>
+                  ))
+                )}
```

Comment:
This is a good solution, but (as is often the case) the developers of antd (the UI library we use) have thought of this use case and included a `loading` prop in the `Select` component. Setting it to `this.state.isFetchingData` will show a loading spinner during loading :) (See the docs for an example and description: https://ant.design/components/select/#Select-props)

Reply:
Nice! Thanks a lot for pointing it out to me :)

Reply:
The loading property does not work, so I used the notFoundContent property instead.

Reply:
Ah, I see, the `loading` property seems to only be available in antd 3.11.0 which we do not have yet. Your solution works very well too :+1: 
I'll create an issue to update antd sometime in the future and include a note about the `loading` property :)